### PR TITLE
Update huygd.json and register huygmd.is-a.dev

### PR DESCRIPTION
**Pull request by huygmdvn**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

huygd.json-https://huygd.is-a.dev/
huygmd.json-https://ltempx.us.kg/(Maybe I will change the source code,this is temporary)

### CODE REVIEW

**Review by Stef-DP — DISMISSED** [review]

File: domains/huygd.json [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  "description": "First domain to study",
   "repo": "https://github.com/huygmdvn/register",
   "owner": {
     "username": "huygmdvn",
     "email": "huynhquochuytkna@gmail.com"
   },
   "record": {
-    "A": ["103.77.240.59"]
+    "A": [
+      "103.77.240.59"
```
[the remark this answers is not in the document]
Comment (huygmdvn) [reply domains/huygd.json:10]: https://huygd.is-a.dev/
Here sir

**Review by iostpa — DISMISSED** [review]
Is `huygmd.json` for a hosting service? (Judging by the link you gave in the PR description)

File: domains/huygd.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (CuteDog5695) [reply domains/huygd.json:10]: Seems good to me

**Comment by huygmdvn** [comment]
I just understand that huygmd.json file use to register huygmd.is-a.dev.With an A record and TXT record.The link in the desc just a temp demo,When the domain approved,I will remove these

**Review by wdhdev — APPROVED** [review]

**Pull request merged**